fenix/app/src/main/java/org/mozilla/fenix/compose/ComposableFactory.kt

```diff
@@ -0,0 +1,75 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.tooling.preview.Preview
+import mozilla.components.concept.base.images.ImageLoadRequest
+import org.mozilla.fenix.components.components
+import org.mozilla.fenix.theme.FirefoxTheme
+import org.mozilla.fenix.compose.ThumbnailImage as ThumbnailImageContent
+
+/**
+ * Factory object used to instantiate Composables with preview-breaking dependencies, such as
+ * [components].
+ */
+object ComposableFactory {
```

Comment:
Certainly open to workshopping the object name

Reply:
❓ I'm not really use if we need this object in the first place, what was the reason for this? From a caller perspectice, I'm more likely to search and write `ThumbnailImage` rather than `Object.ThumbnailImage`